tictactoe.css

```diff
@@ -0,0 +1,19 @@
+#grid {
+	width: 400px;
+	height: 400px;
+	font-size: 70px;
+}
+
+.row {
+	height: 100px;
+	width: 400px;
+	margin: 0;
+}
+
+.tile {
+	width: 100px;
+	height: 100px;
+	background-color: #eeeeee;
+	display: inline-block;
+	text-align: center;
+}
```

Comment:
Try adding a `float: left` value to the `.tile` selector. It'll help with ensuring that the divs don't resize weirdly when a **X** or **O** is placed inside the tiles. Don't forget, you'll want to use `clear` as well if there are any issues with boxes appearing on the wrong lines.